client/landing/jetpack-cloud/sections/auth/connect/index.tsx

```diff
@@ -29,11 +29,11 @@ const Connect: FunctionComponent< Props > = ( { authUrl } ) => {
 				<JetpackLogo full monochrome={ false } size={ 72 } />
 				<p>
 					{ translate(
-						'Welcome to Jetpack Cloud. Authorize with your WordPress.com credentials to get started.'
+						'Welcome to Jetpack. Authorize with your WordPress.com credentials to get started.'
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.